src/server/apsystem.ts

```diff
@@ -93,6 +93,11 @@ export default class ActivityPubSystem {
   async verifySignedRequest (request: FastifyRequest, fromActor?: string): Promise<string> {
     // TODO: Fetch and verify Digest header
     const { url, method, headers } = request
+
+    if (headers.signature === undefined) {
+      return ""
```

Comment:
Instead off returning we should throw an error of some sort. Like a `400` or a `401`, then any code that wants to ignore the error should explicitly catch it and handle it.

Reply:
done!